src/libstd/sys/cloudabi/stdio.rs

```diff
@@ -21,6 +21,10 @@ impl Stdout {
     pub fn new() -> io::Result<Stdout> {
         Ok(Stdout(()))
     }
+
+    pub fn is_tty(&self) -> bool {
+        false
```

Comment:
I think we'll want to conservatively assume we are on a TTY when we don't know how to check.

Reply:
Ah, that was what I meant to do.  I’ll fix it.